app/src/main/java/net/vonforst/evmap/model/ChargersModel.kt

```diff
@@ -24,6 +24,32 @@ import kotlin.math.floor
 
 sealed class ChargepointListItem
 
+/**
+ * A whole charging site (potentially with multiple chargepoints).
+ *
+ * @param id A unique number per charging site
+ * @param dataSource The name of the data source
+ * @param coordinates The latitude / longitude of this charge location
+ * @param address The charge location address
+ * @param chargepoints List of chargepoints at this location
+ * @param network The charging network (Mobility Service Provider, MSP)
+ * @param url A link to this charging site
+ * @param editUrl A link to a website where this charging site can be edited
+ * @param faultReport Set this if the charging site is reported to be out of service
+ * @param verified For crowdsourced data sources, this means that the data has been verified
+ *   by an independent person
+ * @param barrierFree Whether this charge location can be used without prior registration
+ * @param operator The operator of this charge location (Charge Point Operator, CPO)
+ * @param generalInformation TODO
+ * @param amenities TODO
+ * @param locationDescription TODO
```

Comment:
@johan12345 Do you have a good description for these three fields? I was a bit unsure about them.

Reply:
GoingElectric labels these fields as "Allgemeine Information (Informationen, für die es keinen eigenen Punkt gibt)", "Ladeweile" and "Beschreibung der Lage / Position". So maybe something like:
```
* @param generalInformation General information about this charging site that does not fit anywhere else
* @param amenities Description of amenities available at or near the charging site (toilets, food, accommodation, landmarks, etc.)
* @param locationDescription Directions on how to find the charger (e.g. "In the parking garage on level 5")
```

Reply:
Sounds good!